src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Models/DataElement.cs

```diff
@@ -43,10 +44,10 @@ public class Data
         public string StorageUrl { get; set; }
 
         /// <summary>
-        /// path to storage
+        /// Links to access the data elements
         /// </summary>
-        [JsonProperty(PropertyName = "link")]
-        public string Link { get; set; }
+        [JsonProperty(PropertyName = "dataLinks")]
```

Comment:
jsonproperty should probably match property name. 
Set to selfLinks for instances if I'm not mistaken

Reply:
yup. DataLinks it is.